### PR TITLE
#323 RqMultipart.Fake expecting Request as ctor. input

### PR DESCRIPTION
**Pull request by erimerturk**
<code>RqMultipart.Fake </code> accepting Request as ctor. input type. String type version is removed. Issue #323

### CODE REVIEW

**Comment by davvd** [comment]
@erimerturk Thanks for the pull request, let me find a reviewer..

**Comment by davvd** [comment]
@longtimeago please review, thanks

**Comment by longtimeago** [comment]
@erimerturk please, mention task number in PR description. Thank you!

File: src/main/java/org/takes/rq/RqMultipart.java [hunk 1]
```diff
@@ -474,15 +450,26 @@ public InputStream body() throws IOException {
...
+                    .append("Content-Disposition: ")
+                    .append(
+                        new RqHeaders.Smart(
+                            new RqHeaders.Base(each)
+                        // @checkstyle MultipleStringLiteralsCheck (1 line)
+                        ).single("Content-Disposition")
+                    ).append(CRLF);
+                final String body = new RqPrint(each).printBody();
+                if (!(CRLF.equals(body) || "".equals(body))) {
+                    builder.append(String.format("\r\n%s\r\n", body));
```
Comment (longtimeago) [inline src/main/java/org/takes/rq/RqMultipart.java]: @erimerturk maybe we could do this without formatting?
```
builder.append(CRLF).append(body().append(CRLF)
```
It's more natural "builder" way ... we will also reuse CRLF constant

File: src/main/java/org/takes/rq/RqMultipart.java [hunk 2]
```diff
@@ -474,15 +450,26 @@ public InputStream body() throws IOException {
...
+                }
             }
             builder.append("Content-Transfer-Encoding: utf-8\r\n")
```
Comment (longtimeago) [inline src/main/java/org/takes/rq/RqMultipart.java]: @erimerturk same here

**Comment by longtimeago** [comment]
@erimerturk Thank you! Please, see 2 remarks above

**Comment by erimerturk** [comment]
@longtimeago it is ready review, thanks for comments

**Comment by longtimeago** [comment]
@erimerturk thank you!

**Comment by longtimeago** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by erimerturk** [comment]
@yegor256 updated from master, i think it will success now

**Comment by yegor256** [comment]
@rultor merge

**Comment by davvd** [comment]
@ypshenychka please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by ypshenychka** [comment]
@longtimeago 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Two issues were found during code review. Try to find at least three problems while future reviews.

Please confirm that everything is clear and you have no questions regarding this approach.

**Comment by davvd** [comment]
@rultor deploy

**Comment by longtimeago** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@longtimeago Thank you

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks, got it. everybody, please try to make it good next time

**Comment by davvd** [comment]
@longtimeago 10 mins was added to the account of @ypshenychka (for QA review), in transaction 58652336

thank you, added **20 mins** to your acc, payment num is `58652355`, 47 hours and 36 mins spent in total to complete

extra minutes for review comments (c=5)

+20 added to your rating, at the moment it is: [+3867](http://www.netbout.com/b/35024?open=rating)

**Pull request merged**